test/npm_test.dart

```diff
@@ -1022,6 +1070,66 @@ void main() {
       await process.shouldExit(1);
     });
   });
+
+  group("package:cli_pkg/js.dart", () {
+    group("wrapJSException in strict mode", () {
+      // Asserts that the JS [expression] doesn't crash when caught by Dart code
+      // after going through `wrapJSExceptions()`.
```

Comment:
Should this use `///`?

Reply:
No, this is a local variable so DartDoc will ignore it regardless.

Reply:
TIL nested functions don't get dartdoc even for IDEs
https://github.com/dart-lang/sdk/issues/39746